api/src/controllers/publications.js

```diff
@@ -314,11 +314,30 @@ function importPublications(req, res, next) {
     .catch((err) => next(fillErrorObject(500, 'Server error', [err.errors])));
 }
 
+/**
+ * Handles a PATCH request to delete a list of publication by the mongo object id on the endpoint /publications/
+ *
+ * @param req request object - the list of publication ids given in the body
+ * @param res response object
+ * @returns 200: publications deleted successfully
+ * @returns 400: error deleting publications
+ */
+async function deleteBulkPublications(req, res, next) {
```

Comment:
nit with function name. I think `batch` delete is more common than `bulk` delete in terms of naming this kind of functionality (as with the PR name too). Also, curious about why `PATCH` and why not `DELETE`?

Reply:
cause the delete api call only accept a parameter (the deleted document id), but do not accept body